frontend/src/app/features/team-planner/team-planner/planner/team-planner.component.html

```diff
@@ -25,6 +25,19 @@
       data-qa-selector="add-existing-pane"
     >
     </op-add-existing-pane>
+
+    <button
+      class="button button_no-margin op-team-planner--view-select-dropdown"
+      [title]="currentViewTitle"
+      opTeamPlannerViewSelectDropdown
+      [viewOptions]="viewOptions"
+      (viewSelected)="switchView($event)"
+      data-qa-selector="op-team-planner--view-select-dropdown">
+      <span class="button--text ellipsis"
+            [textContent]="currentViewTitle"
+            aria-hidden="true"></span>
+      <op-icon icon-classes="button--icon icon-small icon-pulldown hidden-for-mobile"></op-icon>
```

Comment:
```suggestion
      <span class="spot-icon spot-icon_pulldown hidden-for-mobile"></span>
```

Reply:
This appears to only be working out of the box when also applying `spot-button` to the surrounding button. With some fiddling I can get it to work but the intermediary result:
<img width="288" alt="image" src="https://user-images.githubusercontent.com/617519/201094731-86ff4469-efad-4ad4-b7dd-96ec923ada47.png">


looks way off from the wireframe
<img width="93" alt="image" src="https://user-images.githubusercontent.com/617519/201094518-74c90324-95fd-49c4-8e4b-ea9c95a8ff66.png">

So I would have to add more CSS to basically revert part of what spot-icon does. 

I'd rather wait for `spot-button` to make its way in.

Reply:
All right!